consensus/core/src/network/anemo_network.rs

```diff
@@ -521,6 +579,20 @@ pub(crate) struct FetchBlocksResponse {
     blocks: Vec<Bytes>,
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+pub(crate) struct FetchCommitsRequest {
+    start: CommitIndex,
+    end: CommitIndex,
+}
+
+#[derive(Clone, Serialize, Deserialize)]
+pub(crate) struct FetchCommitsResponse {
+    // Serialized Commit.
+    commits: Vec<Bytes>,
+    // Serialized SignedBlock.
+    certifier_blocks: Vec<Bytes>,
```

Comment:
nit: `signed_blocks`?

Reply:
I want to make it clear that the purpose of the blocks are to certify commits.